pwiz_tools/Skyline/Properties/Resources.resx

```diff
@@ -10077,4 +10077,10 @@ Recognized isotopes include: {2}</value>
   <data name="EditDriftTimePredictorDlg_EditDriftTimePredictorDlg_Adduct" xml:space="preserve">
     <value>Adduct</value>
   </data>
+  <data name="ExportMethodDlg_OkDialog_You_are_missing_compensation_voltages_for_the_following_" xml:space="preserve">
+    <value>You are missing compensation voltages for the following:</value>
+  </data>
+  <data name="ExportMethodDlg_OkDialog_You_can_set_explicit_compensation_voltages_for_these__or_add_their_values_to_a_document_optimization_library_" xml:space="preserve">
+    <value>You can set explicit compensation voltages for these, or add their values to a document optimization library.</value>

```

Comment:
Mention Transition Settings - Prediction tab. Otherwise, I have no idea where to find this "optimization library".